### PR TITLE
Bug/shashank kavli/158#

### LINKED ISSUE #158: [UX] Theme Toggle icon
#### Current Behavior

On mobile devices (small screens) Theme Toggle icon is in the dropdown menu and not visible upfront, imo it would be nice to have it next to the hamburger icon.

#### Expected Behavior

Screenshot_2023-09-26_15-27-00

#### Screenshots/Logs

Currently
Screenshot_2023-09-26_15-26-25

#### Environment

- **Host OS:** Linux

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by shashank-kavli**
**Notes for Reviewers**

This PR fixes #158

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by shashank-kavli** [comment]
@goldfishdolphin Pls check and lmk if any changes are required. Hoping that I've raised the PR correctly 🤞

**Comment by goldfishdolphin** [comment]
Looking good 👍🏽

File: site/src/components/Navigation/Navigation.styles.js [hunk 1]
```diff
@@ -177,9 +177,9 @@ export const Header = styled.header`
         padding: 10px 0;
         display: flex;
         justify-content: center;
-        align-items:center;
-        border-radius:20px;
-        border: 1px solid gray
+        align-items:center
+        // border-radius:20px;
```
Comment (goldfishdolphin) [inline site/src/components/Navigation/Navigation.styles.js:181]:
```suggestion
 
```

File: site/src/components/Navigation/Navigation.styles.js [hunk 2]
```diff
@@ -177,9 +177,9 @@ export const Header = styled.header`
...
+        // border: 1px solid gray
```
Comment (goldfishdolphin) [inline site/src/components/Navigation/Navigation.styles.js:182]:
```suggestion
       
```

File: site/src/components/Navigation/index.js [hunk 3]
```diff
@@ -43,9 +44,9 @@ function Navigation({theme, toggleTheme, showSignUpButton}) {
         </div>        
       </nav>
       <div className={openNav ? "dropdown_menu_opened" : "dropdown_menu_closed"}>
-            <div className="mode-btn">
+            {/* <div className="mode-btn">
```
Comment (goldfishdolphin) [inline site/src/components/Navigation/index.js:47]:
```suggestion

```

File: site/src/components/Navigation/index.js [hunk 4]
```diff
@@ -43,9 +44,9 @@ function Navigation({theme, toggleTheme, showSignUpButton}) {
...
               <Toggle theme={theme} toggleTheme={toggleTheme} />
```
Comment (goldfishdolphin) [inline site/src/components/Navigation/index.js:48]:
```suggestion
         
```

File: site/src/components/Navigation/index.js [hunk 5]
```diff
@@ -43,9 +44,9 @@ function Navigation({theme, toggleTheme, showSignUpButton}) {
...
-            </div>
+            </div> */}
```
Comment (goldfishdolphin) [inline site/src/components/Navigation/index.js:49]:
```suggestion
        
```

**Review by goldfishdolphin — CHANGES_REQUESTED** [review]

**Comment by goldfishdolphin** [comment]
Please remove the commented code. Otherwise all good 👍🏽

**Comment by shashank-kavli** [comment]
@goldfishdolphin I've made the requested changes

**Review by goldfishdolphin — APPROVED** [review]
Thanks @shashank-kavli  for your great work .

**Comment by saurabh100ni** [comment]
Let's discuss this Item in tomorrow's website call, 
please add it as your agenda if you would: 
https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit

**Pull request closed without merging**